### ISSUE TITLE
Specify JSON-LD fields to be translated

### ISSUE THREAD

**Issue by S4mmyb** [regen-network/regen-web#2529]
We need to collect all metadata fields from our JSON-LD columns that should be translated (metadata_graph.metadata and project.metadata).

We can look those up from our legacy repo [GitHub - regen-network/regen-registry-standards: :seedling: RDF and SHACL schemas for Regen Registry](https://github.com/regen-network/regen-registry-standards) for existing metadata and from regen-data-standards repo for terrasos related fields once [refactor Terrasos specific project schema to its own linkml class by clevinson · Pull Request #23 · regen-network/regen-data-standards](https://github.com/regen-network/regen-data-standards/pull/23)  and [Add conservationStatus field to project metadata by wgardiner · Pull Request #20 · regen-network/regen-data-standards](https://github.com/regen-network/regen-data-standards/pull/20)  are merged.

**Comment by S4mmyb** [comment]
@erikalogie can you take a look through this?

**Comment by blushi** [comment]
I don't think we need to translate `regen:marketType`, we don't really display the value directly in the UI but rather use the value to compare with our internal constant, see https://github.com/regen-network/regen-web/blob/6049a683277e0194738ce1e718baa41751f36541/web-marketplace/src/lib/db/types/json-ld/terrasos-project-metadata.ts#L32 and then display relevant logic
If we translate it this means we would need different constants based on the selected language which complicates things unnessarily.

For `regen:SDGs`, we only specify the @id which then allows us to map it to a SDG image in sanity so I don't think we need to translate this either.

Generally, anything that is an `@id` or `@type` shouldn't get translated. So for object fields like `regen:coBenefits`, which has @id, @type, `schema:name` and `regen:SDGs`, did you mean by "Yes" that `schema:name` should be translated in particular?

**Comment by blushi** [comment]
Also for duration like `regen:monitoringFrequency`, how to translate those since there are using ISO 8601 duration format?

**Comment by erikalogie** [comment]
> @erikalogie can you take a look through this?

LGTM aside from Marie's comments

**Comment by blushi** [comment]
@S4mmyb Here's an updated list of json fields to be translated. When we need to translate a value within a nested object, I separate the keys with a dot, this is how our translation process works. This way we can translate `schema:name` within `approvedMethodologies` for instance but not all the other `schema:name` values. So do you think we still want to translate all top level `schema:name`? like project or credit class names...

Within `verificationReports` it looks like we only store some `schema:url` so I excluded those. As well as any frequency/duration related fields which uses ISO 8601 format.

I've also included some fields that were listed but should be translated IMO:
- description of all project stakeholders
- some project page metadata fields

```
"schema:name", ???
"schema:description",
"regen:bioregion",
"regen:biomeType",
"regen:conservationStatus",
"regen:ecosystemType",
"regen:environmentType",
"regen:environmentalAuthority.schema:name",
"regen:managementAreas.regen:projectActivity",
"regen:projectDeveloper.schema:description",
"regen:projectMonitor.schema:description",
"regen:projectOperator.schema:description",
"regen:projectOwner.schema:description",
"regen:projectVerifier.schema:description",
"regen:subWatershed",
"regen:watershed",
"regen:sectoralScope",
"regen:certifications",
"regen:projectActivities",
"regen:verificationMethod",
"regen:approvedMethodologies.schema:itemListElement.schema:name",
"regen:offsetGenerationMethod",
"regen:offsetProtocol.schema:name",
"regen:projectType",
"regen:measuredGHGs",
"regen:leakageApproach",
"regen:landOwnershipType",
"regen:bufferPoolAccounts.schema:name",
"regen:eligibleActivities",
"regen:verificationMethods",
"regen:additionalityApproach",
"regen:additionalityAssessment",
"regen:creditGenerationMethod",
"regen:biocharApplicationType",
"regen:baselineMethod",
"regen:leakageAssessment",
"regen:geographicApplicability",
"regen:primaryImpact.schema:name",
"regen:biocharFeedstockMaterial",
"regen:additionalCertifications.schema:name",
"regen:supplementalDatasets.schema:name"

"regen:story",
"regen:storyTitle",
"regen:previewPhoto.schema:creditText",
"regen:galleryPhotos.schema:caption",
"regen:galleryPhotos.schema:creditText",
"schema:creditText",
"regen:storyMedia.schema:creditText"
```

**Comment by S4mmyb** [comment]
Hey @blushi, thanks for updating that. I was thinking about this more and I think To enhance accessibility, we recommend translating all `schema:name` values, covering names of projects, methodologies, standards, reports, and co-benefits. My rationale is that google translate will probably leave unique project names, organization names, and standards (e.g., “ICROA”) t untranslated which allows us to preserve brand integrity and recognition, while translating descriptive names (e.g., methodologies, reports, co-benefits) to ensure international users can fully understand the content and context.

I think we try that and see how it works, then for the edge cases we can always go back and update the ones it didn't do well?

**Comment by blushi** [comment]
So for "Verified Carbon Standard" credit class, I've done some tests and it will translate it as "Estándar de carbono verificado", is that fine?

**Comment by blushi** [comment]
I wasn't sure about `additionalCertifications` either, we have for instance the following in some batch metadata:
```
'regen:additionalCertifications': [
        {
          'schema:name': 'CCB-Gold',
          'schema:url': {
            '@type': 'schema:URL',
            '@value': 'https://verra.org/project/ccb-program/',
          },
        },
      ]
```

If we translate the schema:name in it, it will give "CCB-Oro"
